contributors/design-proposals/kubelet-eviction.md

```diff
@@ -425,6 +425,22 @@ from placing new best effort pods on the node since they will be rejected by the
 On the other hand, the `DiskPressure` condition if true should dissuade the scheduler from
 placing **any** new pods on the node since they will be rejected by the `kubelet` in admission.
 
+## Enforcing Node Allocatable
+
+To enforce [Node Allocatable](./node-allocatable.md), Kubelet primarily uses cgroups.
+However `storage` cannot be enforced using cgroups.
+
+Once Kubelet supports `storage` as an `Allocatable` resource, Kubelet will perform evictions whenever the total storage usage by pods exceed node allocatable.
+
+The trigger threshold for storage evictions will not be user configurable for the purposes of `Allocatable`.
+Kubelet will evict pods once the `storage` usage is greater than or equal to `Allocatable`.
+Kubelet will evict pods until it can reclaim `5%` of `storage Allocatable`, thereby brining down usage to `95%` of `Allocatable`.
+These thresholds apply for both storage `capacity` and `inodes`.
+
+*Note that these values are subject to change based on feedback from production.*
```

Comment:
My two cents is that this text is premature pending agreement on the local storage proposal.  Can we remove this text and just state in the future storage will be a part of allocatable.  I am not able to agree at this time on lack of configuration and it's also not a planned 1.6 feature.